assets/eip-5218/ic3license/ic3license.tex

```diff
@@ -201,11 +237,16 @@ \section{License Text}
 		
 	\item	A ``Ledger'' is a blockchain, database, or other digital system that records information about Unique Identifiers.
 		
+	\item	An ``NFT'' is a Unique Identifier in a Ledger.
+		
 	\item	A ``Private Key'' is a cryptographic key, the use of which is necessary to modify the information about a Unique Identifier recorded in a Ledger.
 		
-	\item	An ``NFT'' is a Unique Identifier in a Ledger that is Associated with a person or entity when that person or entity has substantially exclusive control over the Private Key necessary to modify the information about that Unique Identifier in that Ledger.
+	\item An NFT  is ``Associated'' with a person or entity when that person or entity has substantially exclusive control over the Private Key necessary to modify the information about that Unique Identifier in that Ledger.
 		
-	\item	The ``Controller'' of an NFT is the person or entity with whom the NFT is Associated.
+	\item The ``Controller'' of an NFT is the person or entity with whom the NFT is Associated.
+	
+	\item An ``NFT Generator'' is a smart contract, function, or other service that creates NFTs in a Ledger.
```

Comment:
[Editorial] I think this is a bit unclear
- A smart contract following ERC-721 or ERC-1155 when deployed offers NFT feature, it can be referred to as "that creates NFT"
- A deployer service or smart contract that deploys said smart contract can also be referred to as "that creates NFT"
- A person or machine that operates such deployer service can also be referred to as "that creates NFT"

Consider further clarify which one do you refer to?